backend/accounts.go

```diff
@@ -82,3 +88,72 @@ func (backend *Backend) SupportedCoins(keystore keystore.Keystore) []coinpkg.Cod
 	}
 	return availableCoins
 }
+
+// createAndPersistAccountConfig adds an account for the given coin and account number. The account numbers start
+// at 0 (first account). The added account will be a unified account supporting all types that the
+// keystore supports. The keypaths will be standard BIP44 keypaths for the respective account types.
+func (backend *Backend) createAndPersistAccountConfig(coinCode coinpkg.Code, accountNumber uint16, keystore keystore.Keystore) error {
+	coin, err := backend.Coin(coinCode)
+	if err != nil {
+		return err
+	}
+
+	// The first account's code does not have a suffix, to keep the same code as before we introduced
+	// multiple accounts. This is to not invalidate the account cache the user had before.
+	accountCode := string(coinCode)
+	if accountNumber > 0 {
+		accountCode = fmt.Sprintf("%s-%d", coinCode, accountNumber)
+	}
+	log := backend.log.
+		WithField("accountCode", accountCode).
+		WithField("coinCode", coinCode).
+		WithField("accountNumber", accountNumber)
+
+	// This used to be a user-facing setting. Now we simply use it for migration to decide which
+	// coins to add by default. Only for the first account.
+	if accountNumber == 0 && !backend.config.AppConfig().Backend.CoinActive(coinCode) {
```

Comment:
just realized this will not work, e.g. if someone starts with only bitcoin, and then e.g. wants to add the first litecoin account. i will move this to the callsite when adding the initial default accounts.